test-automation-selenide/README.md

```diff
@@ -0,0 +1,50 @@
+# Test-automation-selenide
+
+### Setup
+
+Import module to your project:
+   * Maven - add the dependency to your project pom.xml:
+   
+   `<dependency> 
+     <groupId>com.nortal.test</groupId>
+     <artifactId>test-automation-selenide</artifactId>
+     <version>${test.framework.version}</version>
+   </dependency>`
+
+   * Gradle - add the dependency to your project build.gradle.kts:
+   
+   `implementation(":test-automation-selenide")`
```

Comment:
it should be intTestImplementation("com.nortal.test:test-automation-selenide:${testAutomationFrameworkVersion}")